content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+can see the effect of it by looking at the diff.
+
+```dvc
+$ git diff params.yaml
+```
+
+```git
+-model:
+-  conv_units: 16
++model:
++  conv_units: 24
+```
+
+## 🏃‍♂️🏃🏾‍♂️🏃🏻‍♂️ Run multiple experiments in parallel
+
+Instead of running the experiments one-by-one, we can define them to run in a
+batch. This is especially handy when you have long running experiments.
+
+We add experiments to the queue using the `--queue` option of `dvc exp run`. We
+also use `-S` (`--set-param`) to set a value for the parameter.
+
+```dvc
+$ dvc exp run --queue -S model.conv_units=32
+Queued experiment '6518f17' for future execution.
+$ dvc exp run --queue -S model.conv_units=64
+Queued experiment '30eb9b2' for future execution.
+$ dvc exp run --queue -S model.conv_units=128
+Queued experiment 'ac66940' for future execution.
+$ dvc exp run --queue -S model.conv_units=256
+Queued experiment '8bb6049' for future execution.
+```
+
+Next, run all (`--run-all`) queued experiments in parallel. You can specify the
+number of parallel processes using `--jobs`:
+
+```dvc
+$ dvc exp run --run-all --jobs 2
+```
+
+## ↔️ Comparing experiments
+
+The experiments are run several times with different parameters. We use
+`dvc exp show` to compare all of these experiments. This command presents the
+parameters and metrics produced in experiments in a nicely formatted table.
+
+```dvc
+$ dvc exp show
+```
+
+![](/img/start-dvc-exp-show-210704.png)
+
+By default it shows all the parameters and the metrics with the timestamp. If
+you have large number of parameters, metrics or experiments, this may lead to a
+cluttered view. You can limit the table to specific metrics, or parameters, or
+hide the timestamp column with `--include-metrics`, `--include-params`, or
+`--no-timestamp` options of the command, respectively.
+
+```dvc
+$ dvc exp show --no-timestamp \
+  --include-params model.conv_units --include-metrics acc
+```
+
+![](/img/start-dvc-exp-show-no-timestamp-210704.png)
+
+## 🔏 Persisting experiments
+
+After selecting an experiment from the table, you can commit the hyperparameters
+and other dependencies that produced this successful experiment to your Git
+history.
+
+`dvc exp apply` brings back all specific artifacts and parameters from the
+experiment to the <abbr>workspace</abbr>.
+
+```dvc
+$ dvc exp apply exp-cb13f
+Changes for experiment 'exp-cb13f' have been applied to your current workspace.
+```
+
+We can see the changes in the repository and commit them to Git.
+
+```dvc
+$ git diff
+$ git add .
+$ git commit -m "Successful experiment"
+```
+
+It may also be desirable to commit a particular experiment to a Git branch
+directly, without bringing to the workspace.
+
+```dvc
+$ dvc exp branch exp-05e87 "cnn-256"
+Git branch 'cnn-256' has been created from experiment 'exp-05e87'.
+To switch to the new branch run:
+
+        git checkout cnn-256
+```
+
+You can then checkout and continue from working this branch as usual.
+
+## 📛 Note on experiment names
```

Comment:
It think this it a bit too much - to have the whole section like this as a Note. Is there a very important reason to make it of the same importance as "running experiments" and other sections?

Can we hide it?

Reply:
The reason I put this into a separate section is @aguschin's comments about experiments names, I think in iterative/dvc#6050 . For a newcomer having two names, one reported by `--queue`, one given after the run is a bit confusing and I'd rather have a single autogenerated commit name for both of these. Current behavior is a bit confusing for the newcomer. 

I can move this to a relevant UG page and hide here.

Reply:
This is in the current dvc sprint to fix, and I just updated the importance. Let's assume this will be fixed before publication, and we can bump the priority further if needed. Hopefully, we can hide or drop this whole section or at least greatly simplify.

Reply:
cc @karajan1001. If you have capacity, please pick up this issue next.

Reply:
removed the naming section